fdbserver/BlobManager.actor.cpp

```diff
@@ -318,6 +318,11 @@ ACTOR Future<Void> doRangeAssignment(BlobManagerData* bmData, RangeAssignment as
 			req.managerEpoch = bmData->epoch;
 			req.managerSeqno = seqNo;
 			req.continueAssignment = assignment.assign.get().continueAssignment;
+
+			// if that worker isn't alive anymore, add the range back into the stream
+			if (bmData->workersById.count(workerID) == 0) {
+				throw worker_for_granule_not_found();
```

Comment:
while it's nice to have a more specific error here, since we're only using this error for flow control, it is probably better to reuse an existing error, since we don't have infinite error code space. Maybe no_more_servers or a generic operation_failed or something?

Reply:
Sounds good. will re-use one